input/resources/au-core-actor-requester.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ActorDefinition xmlns="http://hl7.org/fhir">
+  <id value="au-core-actor-requester"/>
+  <url value="http://hl7.org.au/fhir/core/ActorDefinition/au-core-actor-requester" />
+  <name value="AUCoreRequester"/> 
+  <title value="AU Core Requester"/> 
+  <status value="draft"/>
+  <description value="The AU Core Requester is a system that responds to the data access request submitted by requesters, providing responses to requests for core digital health and administrative information."/> 
```

Comment:
"The AU Core Requester is a system that **responds**...." - correct to state this is a system that creates and initiates a data access request.

Reply:
fixed.